.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+services:
+  - docker
+
+jobs:
+  include:
+    - name: "Tests: Writer"
+      python: "3.8"
+      env:
+        - COMPOSE_FILE=writer/dc.test.yml
+      script:
```

Comment:
First, I was thinking about a test_entrypoint script or so, but single staps are better for travis. So it seems fine :)

Reply:
Nice to see that you went through the same thoughts as me :D